src/internal/operations/backup.go

```diff
@@ -655,30 +657,37 @@ func (op *BackupOperation) persistResults(
 // stores the operation details, results, and selectors in the backup manifest.
 func (op *BackupOperation) createBackupModels(
 	ctx context.Context,
-	detailsStore streamstore.Writer,
+	detailsStore, errorsStore streamstore.Writer,
 	snapID string,
 	backupID model.StableID,
 	backupDetails *details.Details,
 ) error {
 	ctx = clues.Add(ctx, "snapshot_id", snapID)
+	errs := op.Errors
 
 	if backupDetails == nil {
 		return clues.New("no backup details to record").WithClues(ctx)
 	}
 
-	detailsID, err := detailsStore.Write(ctx, backupDetails, op.Errors)
+	detailsID, err := detailsStore.Write(ctx, backupDetails, errs)
 	if err != nil {
-		return clues.Wrap(err, "creating backupDetails model").WithClues(ctx)
+		return clues.Wrap(err, "creating backupDetails persistence").WithClues(ctx)
+	}
+
+	errorsID, err := errorsStore.Write(ctx, errs.Errors(), errs)
```

Comment:
both writing and possibly adding to `errs` is probably a bad idea. What if we serialized things as we went along and somehow we ended up adding to `errs` in the call to persist `errs`?

Reply:
`errs.Errors()` is a wholly separate snapshot (not kopia) of data relative to the errs Bus.  Worst case scenario is we write to `errs` within the streamStore, and don't persist that error.  Not too different from your standard "persist an error with a client that can error" problem.

Reply:
>errs.Errors() is a wholly separate snapshot (not kopia) of data relative to the errs Bus.

That assumes a very specific implementation of `errs.Errors()` (specifically the current one that makes a snapshot of the already existing errors). Separating where errors are collected during the call from what errors are being serialized is a more future-proof solution because changing how `errs.Errors()` is marshalled won't cause unexpected failures here